### ISSUE TITLE
@3x icons?

### ISSUE THREAD

**Issue by pdeitel** [0xced/iOS-Artwork-Extractor#27]
I might be mistaken, but I think the iPhone 6 plus simulator has @3x icons. 

Is there a way to get these?

**Comment by aentan** [comment]
I need them too.

**Comment by ide** [comment]
@0xced: any interest in my PR?

**Comment by johnqh** [comment]
Would like 3X too.

**Comment by nuthatch** [comment]
Ide's @3x branch works. Thank you!

**Comment by Stunner** [comment]
Yes, Ide's 3x branch works: https://github.com/ide/iOS-Artwork-Extractor/tree/3x Thanks for the tip @nuthatch

**Comment by malhal** [comment]
Shouldn't this have been merged to the trunk?
